examples/image-streams/image-streams-centos7.json

```diff
@@ -14,7 +14,11 @@
         "dockerImageRepository": "openshift/ruby-20-centos7",
```

Comment:
Hrm - that may actually be what we want.  We only want to import tags we
whitelist for openshift customers.

On Tue, Oct 27, 2015 at 12:43 PM, Ben Parees <notifications@github.com>
wrote:

> In examples/image-streams/image-streams-centos7.json
> <https://github.com/openshift/origin/pull/5409#discussion_r43149082>:
>
> > @@ -14,7 +14,11 @@
> >          "dockerImageRepository": "openshift/ruby-20-centos7",
>
> yeah i was debating whether to leave that or not... if i remove it,
> nothing (ie tags we don't specify explicitly) will get automatically
> imported, i'm not sure that's what we want. so i think it's appropriate to
> leave it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/5409/files#r43149082>.
>